packages/tasks/src/model-libraries.ts

```diff
@@ -298,6 +298,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		repoName: "mindspore",
 		repoUrl: "https://github.com/mindspore-ai/mindspore",
 	},
+	mamba_ssm: {
```

Comment:
just `mamba` would be cleaner, branding-wise?

```suggestion
	mamba: {
```

Reply:
and it's `pip install mamba-ssm` on PyPI so maybe a better option too

I think i'd go with `mamba` personally

Reply:
We discussed this with @osanseviero. The problem of simply `mamba` is that it's a `transformers` architecture as well so it might create some conflicts. Also having `mamba_ssm` as library_name and `mamba` as tag helps with code snippets (to import `Mamba` class).

Reply:
We could have `Mamba` as prettyName but it might be misleading

Reply:
ok but then dash rather than underscore no?

Reply:
nvm let's not complicate stuff, LGTM as is :)

Reply:
good point, addressed in https://github.com/huggingface/huggingface.js/pull/802/commits/e3e7e62c2bd3b0309e1e3a9a203abc430e17c98c and https://github.com/state-spaces/mamba/pull/471/commits/961eccb89f92e34459ea3e6959081c833bba19c3